src/devinterp/slt/forms.py

```diff
@@ -0,0 +1,220 @@
+import matplotlib.pyplot as plt
+from itertools import combinations
+import numpy as np
+import warnings
+
+from devinterp.slt.pca import *
+
+
+def get_osculating_circles(smooth_pc1, smooth_pc2, OSCULATE_START, OSCULATE_END):
```

Comment:
Is there a reason for caps here? Or is this a vestige of the original notebooks?